src/ai/backend/manager/config.py

```diff
@@ -465,11 +465,11 @@ def load(config_path: Path = None, debug: bool = False) -> LocalConfig:
     config.override_with_env(
         raw_cfg, ("docker-registry", "ssl-verify"), "BACKEND_SKIP_SSLCERT_VALIDATION"
     )
-    if debug:
-        config.override_key(raw_cfg, ("debug", "enabled"), True)
-        config.override_key(raw_cfg, ("logging", "level"), "DEBUG")
-        config.override_key(raw_cfg, ("logging", "pkg-ns", "ai.backend"), "DEBUG")
-        config.override_key(raw_cfg, ("logging", "pkg-ns", "aiohttp"), "DEBUG")
+
+    config.override_key(raw_cfg, ("debug", "enabled"), log_level == "debug")
```

Comment:
```suggestion
    config.override_key(raw_cfg, ("debug", "enabled"), log_level == LogSeverity.DEBUG)
```

Reply:
I didn't change this part because log_level is raw string in **load** function in this file.